CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### v2.1.0
+
+* Navigation removed from mapbox-java and moved to separate repo [#439](https://github.com/mapbox/mapbox-java/pull/439)
```

Comment:
Let's link to the new repo, that will be ready shortly after this release: https://github.com/mapbox/mapbox-navigation-android. For example: `The navigation SDK has been removed from this repository into its own, available on [link]`